src/chk/chk.pb-c.c

```diff
@@ -211,37 +211,50 @@ static const ProtobufCFieldDescriptor chk__check_report__field_descriptors[15] =
     0,NULL,NULL    /* reserved1,reserved2, etc */
   },
   {
-    "actions",
+    "act_choices",
     14,
     PROTOBUF_C_LABEL_REPEATED,
     PROTOBUF_C_TYPE_ENUM,
-    offsetof(Chk__CheckReport, n_actions),
-    offsetof(Chk__CheckReport, actions),
+    offsetof(Chk__CheckReport, n_act_choices),
+    offsetof(Chk__CheckReport, act_choices),
     &chk__check_inconsist_action__descriptor,
     NULL,
     0 | PROTOBUF_C_FIELD_FLAG_PACKED,             /* flags */
     0,NULL,NULL    /* reserved1,reserved2, etc */
   },
   {
-    "details",
+    "act_details",
     15,
     PROTOBUF_C_LABEL_REPEATED,
     PROTOBUF_C_TYPE_STRING,
-    offsetof(Chk__CheckReport, n_details),
-    offsetof(Chk__CheckReport, details),
+    offsetof(Chk__CheckReport, n_act_details),
+    offsetof(Chk__CheckReport, act_details),
+    NULL,
+    &protobuf_c_empty_string,
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)